code-challenges.test.js

```diff
@@ -13,47 +13,152 @@
+//INPUT: randomNouns1 array and randomNouns2 array
+//OUTPUT:  randomNouns1 and 2 arrays capitalized
 // a) Create a test with expect statements for each of the variables provided.
 
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+// const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+//  // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
-// b) Create the function that makes the test pass.
+// // b) Create the function that makes the test pass.
 
-// --------------------3) Create a function that takes in a string and logs the index of the first vowel.
+// const arrayCap = (array) => {
+//     return array.map(value => {
+//         return value.charAt().toUpperCase() +value.slice(1)
+//     })
+// }
 
-// a) Create a test with expect statements for each of the variables provided.
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
 
-const vowelTester1 = "learn"
-// Expected output: 1
-const vowelTester2 = "academy"
-// Expected output: 0
-const vowelTester3 = "challenges"
-// Expected output: 2
 
-// b) Create the function that makes the test pass.
+
+//  describe("arrayCap", () => {
+//      it("returns a statement with the appropriate coffee order", () => {
+//        expect(arrayCap(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+//        expect(arrayCap(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+//      })
+//    })
+   // ReferenceError: arrayCap is not defined
+
+
+ // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
+
+ //PSEUDOCODE: Create a const vowelsA labeling the vowels to be searched by index
+ //             Create the function indexVowel that will go through each index to use .include
+//              to find the vowel and give the index as the return
+// INPUT: voweltesters given
+//OUTPUT: 1,0,2
+// a) Create a test with expect statements for each of the variables provided.
+
+ const vowelTester1 = "learn"
+// // Expected output: 1
+ const vowelTester2 = "academy"
+// // Expected output: 0
+ const vowelTester3 = "challenges"
+// // Expected output: 2
+
+// // b) Create the function that makes the test pass.
+const vowelsA = ["a", "e", "i", "o", "u"]
+const indexVowel = array => {
+//     return value.length (value => {
+//         return value.indexOf() .includes(vowelsA => {
+//             return array})
+//         })
+// }
+
```

Comment:
I like that you are attempting different ways. This is an excellent example of where pseudocode will really help you think each step and understand if you are missing something. Things like understanding what .include will produce or if .length will be useful.